askama_shared/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "askama_shared"
-version = "0.12.2"
+version = "0.13.0"
```

Comment:
Why do we need this bump?

Reply:
I un-exported a bunch of symbols. Older `askama_shared` versions would break with this update. Every item in https://github.com/djc/askama/blob/main/askama_derive/src/lib.rs#L8-L10 is gone after the PR.